libyul/AssemblyStack.cpp

```diff
@@ -217,6 +217,29 @@ MachineAssemblyObject AssemblyStack::assemble(Machine _machine) const
 	return MachineAssemblyObject();
 }
 
+namespace
+{
+
+// Duplicated from libsolidity/codegen/CompilerContext.cpp
+// TODO: refactor and remove duplication
```

Comment:
We could just add a member function on `solidity::frontend::OptimiserSettings` for converting to the libevmasm struct.

Reply:
I think `evmasm::Assembly` would be a better place.

Reply:
Done. It's now in `evmasm::Assembly::OptimiserSettings.translate()` and `CompilerContext` uses that now too.